pkg/testutils/mock_config.go

```diff
@@ -14,47 +13,27 @@ import (
 )
 
 const (
-	configMapName = "kubevirt-config"
-	namespace     = "kubevirt"
+	namespace = "kubevirt"
 )
 
-func NewFakeClusterConfig(cfgMap *v1.ConfigMap) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer, cache.SharedIndexInformer) {
-	return NewFakeClusterConfigWithCPUArch(cfgMap, runtime.GOARCH)
-}
-
-func NewFakeClusterConfigWithCPUArch(cfgMap *v1.ConfigMap, CPUArch string) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer, cache.SharedIndexInformer) {
-	configMapInformer, _ := NewFakeInformerFor(&v1.ConfigMap{})
-	crdInformer, _ := NewFakeInformerFor(&extv1.CustomResourceDefinition{})
-	kubeVirtInformer, _ := NewFakeInformerFor(&KVv1.KubeVirt{})
-
-	if cfgMap != nil {
-		copy := copy(cfgMap)
-		configMapInformer.GetStore().Add(copy)
-	}
-
-	AddDataVolumeAPI(crdInformer)
-
-	return virtconfig.NewClusterConfigWithCPUArch(configMapInformer, crdInformer, kubeVirtInformer, namespace, CPUArch), configMapInformer, crdInformer, kubeVirtInformer
-}
-
-func NewFakeClusterConfigUsingKV(kv *KVv1.KubeVirt) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer, cache.SharedIndexInformer) {
+func NewFakeClusterConfigUsingKV(kv *KVv1.KubeVirt) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer) {
 	return NewFakeClusterConfigUsingKVWithCPUArch(kv, runtime.GOARCH)
 }
 
-func NewFakeClusterConfigUsingKVWithCPUArch(kv *KVv1.KubeVirt, CPUArch string) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer, cache.SharedIndexInformer) {
+func NewFakeClusterConfigUsingKVWithCPUArch(kv *KVv1.KubeVirt, CPUArch string) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer) {
 	kv.ResourceVersion = rand.String(10)
-	configMapInformer, _ := NewFakeInformerFor(&v1.ConfigMap{})
+	kv.Status.Phase = "Deployed"
```

Comment:
@nunnatsa Can you elaborate on this initialization? I see it wasn't there before

Reply:
Right. Some unit test failed. These test were based on a configMap configuration, but in this PR they are using KV configuration. The problem is in this code https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-config/config-map.go#L599-L601
The new configuration for the test are ignored, because the `phase` field is not set. Many test that uses KV configurations set the phase field within the test. Here we're ding it in one place for all the test. See https://github.com/kubevirt/kubevirt/blob/931a48c4c326d5f8e5799bc8e8ad02d3933157bc/pkg/testutils/mock_config.go#L67 in the same file, where the same solution is already in use.